spec/middlewares/request_logger_spec.rb

```diff
@@ -7,7 +7,10 @@
   let(:logger) { instance_double(Logger) }
 
   before do
-    allow(Time).to receive(:now).and_return(1, 2)
+    allow(Time).to receive(:now).and_return(
+      Time.new(2022, 12, 19, 0, 0, 0),
+      Time.new(2022, 12, 19, 0, 0, 1)
+    )
```

Comment:
Tempfile needs a real `Time.now` to create a tempfile when parsing uploaded files, so using this fake data instead worked for me.